src/main/liquibase/liquibase-CIEL-data.xml

```diff
@@ -5,7 +5,7 @@
     http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
 
 	<changeSet id="20120315-1000" author="standalone" dbms="mysql">
-		<comment>Preparing schema for adding mvp data</comment>
```

Comment:
Add before the `<comment>` here:

```xml
<validChecksum>3:dc136a76a36273b5cdb2f47b0eb24522</validChecksum>
```

This will prevent the change from being re-executed on upgrade